pkg/sql/parsers/dialect/mysql/mysql_sql.y

```diff
@@ -5603,6 +5749,10 @@ reserved_keyword:
 |   FULLTEXT
 |   FOREIGN
 |	ROW
+|   OUTFILE
```

Comment:
reserved_keyword should refer to https://dev.mysql.com/doc/refman/8.0/en/keywords.html